### ISSUE TITLE
[1.12.2] (Question) Custom Blocks effects/sounds looping.

### ISSUE THREAD

**Issue by Yldales** [OreCruncher/DynamicSurroundings#651]
Hi,

In a DynamicSurroundings's external configuration file, in the _blocks_ object, is it possible to make a sound loop (like the `background`/`periodic` soundType for Biomes sounds) ? Using `chance: 1`, defined sounds are frequently overlapping.

### Tests
```json
  "blocks": [
    {
      "blocks": [
        "chisel:technical1[variation=1]"
      ],
      "chance": 1,
      "sounds": [
        {
          "sound": "minecraft:ambient.amb_fan_slow_lp_01",
          "volume": 0.1,
          "pitch": 1,
          "variable": false
        }
      ]
    }
  ],
```

### Details
- **Mod Version:** 1.12.2-3.6.1.0
- **Forge Version:** 14.23.5.2847
- **Link to client log:** None.
- **Link to crash log:** None.

Thanks in advance for your reply.

**Comment by OreCruncher** [comment]
This isn't possible with DS 1.12.2.  Mojang finally fixed sound looping in 1.16.4, and this is a feature I do plan on adding.   Don't think it would be back ported though considering it requires some fixes in MC itself.

**Comment by Yldales** [comment]
> 
> 
> This isn't possible with DS 1.12.2. Mojang finally fixed sound looping in 1.16.4, and this is a feature I do plan on adding. Don't think it would be back ported though considering it requires some fixes in MC itself.

Hi,

Thank you for these clarifications.
Good luck for the future of this wonderful mod 👌 .
